site/static/shacl-extensions.html

```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Eclipse RDF4J SHACL Extensions</title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove" defer></script>
+    <script class="remove">
+        var respecConfig = {
+            specStatus: "unofficial",
+            editors: [
+                { name: "Håvard Ottestad" },
+            ],
```

Comment:
We should add more editors. Probably Vladimir if he wants to be listed.

Reply:
Not sure if this is the place to do it, but I notice that the current rendered spec links to `https://eclipse.github.io/rdf4j/` at the top (under "latest editor's draft"). Can we adjust that to link to the correct page?

Reply:
Fixed this